stdlib/public/TensorFlow/Execution.swift

```diff
@@ -20,24 +20,27 @@
 // function.  We need to inline if a callee contains tensor ops, not only if
 // it takes and returns a TensorFlow value.
 @_transparent
+@available(*, deprecated)
 public func enableTPU(serverAddress: String? = nil, infeed: Bool = true) {
   _RuntimeConfig.executionMode = .tpu
   if let serverAddress = serverAddress {
     _RuntimeConfig.session = .remote(serverDef: serverAddress)
   }
-  #tfop("tfc.configureTPU", enableInfeed: infeed) as Void
+//   #tfop("tfc.configureTPU", enableInfeed: infeed) as Void
```

Comment:
Why not remove this and the following too?

Reply:
Done.